.pre-commit-config.yaml

```diff
@@ -37,5 +37,6 @@ repos:
           # https://github.com/seanmonstar/pretty-env-logger/pull/49
           # https://github.com/eminence/terminal-size/pull/49
           "-Aclippy::multiple-crate-versions",
+          "-Aclippy::doc-markdown",
```

Comment:
@jpmckinney I had to add this again so that I can commit. We can compare our rust/cargo versions this time.

Reply:
Can you run `rustc --version` and `cargo --version` ?

Reply:
rustc 1.78.0-nightly (b6d2d841b 2024-03-05)
cargo 1.78.0-nightly (f772ec022 2024-03-01)

Reply:
And `cargo clippy --version` (added later) ?

Reply:
clippy 0.1.78 (b6d2d84 2024-03-05)

Reply:
Okay, I can reproduce now.